web/src/components/App.js

```diff
@@ -24,12 +25,15 @@ const App = () => (
     </Toolbar>
 
     <Container is="main">
-      <Route exact path="/" component={Demo} />
-      <Route path="/activities-start" component={ActivitiesStart} />
-      <Route path="/apd-overview" component={ApdOverview} />
-      <Route path="/state-start" component={StateStart} />
-      <Route path="/state-contacts" component={StateContacts} />
-      <Route path="/state-personnel" component={StatePersonnel} />
+      <Switch>
```

Comment:
This is a cool component.  Having just read a little about it, I kind of feel like maybe `<Switch>` should always be at the root of page-level routing.  😛